src/public/media.ts

```diff
@@ -141,6 +142,47 @@ export namespace media {
         callback(invalidInput, null);
         return;
       }
+      // Call the new get media implementation via callbacks if the client version is greater than or equal to '2.0.0'
+      if (callGetMediaViaCallback('2.0.0')) {
+        this.getMediaViaCallback(callback);
+      } else {
+        this.getMediaViaHandler(callback);
+      }
+    }
+
+    private getMediaViaCallback(callback: (error: SdkError, blob: Blob) => void): void {
+      const helper: MediaHelper = {
+        mediaMimeType: this.mimeType,
+        assembleAttachment: [],
+      };
+      const params = [this.content];
+      const actionName = sendMessageRequestToParent('getMedia', params);
```

Comment:
Please rename `actionName` to `messageId` since that's what the `sendMessageRequestToParent` method is actually returning here.

Reply:
done